### ISSUE TITLE
Required ENV_SURICATA_ARGS?

### ISSUE THREAD

**Issue by alphaDev23** [StamusNetworks/suricata-docker#3]
What are the required ENV_SURICATA_ARGS?  There is no description in this repository.  If left blank, the following error is exhibited upon brining up the container:

suricata    | Error: Format string '/usr/bin/suricata\n-c /etc/suricata/suricata.yaml\n--pidfile /var/run/suricata.pid\n%(ENV_SURICATA_ARGS)s' for 'program:suricata.command' contains names ('ENV_SURICATA_ARGS') which cannot be expanded. Available names: ENV_HOME, ENV_HOSTNAME, ENV_INTERFACE, ENV_PATH, group_name, here, host_node_name, process_num, program_name in section 'program:suricata' (file: '/etc/supervisor/conf.d/suricata.conf')
suricata    | For help, use /usr/bin/supervisord -h
suricata exited with code 2

**Comment by pevma** [comment]
from what i see it is called up here - https://github.com/StamusNetworks/suricata-docker/blob/master/supervisor.d/suricata.conf

**Comment by alphaDev23** [comment]
Yes, but what are the required ENV_SURICATA_ARGS?  If left blank, the error in the initial post is exhibited upon bringing up the container.

**Comment by alphaDev23** [comment]
Any update on this?

**Comment by alphaDev23** [comment]
Any thoughts on the issue?

**Comment by pevma** [comment]
Can you paste the full output of the run please (stdout/anything)?

**Comment by alphaDev23** [comment]
Given that ENV_SURICATA_ARGS are not set, the following docker log is not surprising.  What are the required ENV_SURICATA_ARGS?

suricata    | Error: Format string '/usr/bin/suricata\n-c /etc/suricata/suricata.yaml\n--pidfile /var/run/suricata.pid\n%(ENV_SURICATA_ARGS)s' for 'program:suricata.command' contains names ('ENV_SURICATA_ARGS') which cannot be expanded. Available names: ENV_HOME, ENV_HOSTNAME, ENV_INTERFACE, ENV_PATH, group_name, here, host_node_name, process_num, program_name in section 'program:suricata' (file: '/etc/supervisor/conf.d/suricata.conf')
suricata    | For help, use /usr/bin/supervisord -h
suricata exited with code 2

**Comment by pevma** [comment]
Not sure for this one - @regit any input ?
